crates/stark-backend/src/air_builders/debug/check_constraints.rs

```diff
@@ -87,14 +68,14 @@ pub fn check_constraints<R, SC>(
                 RowMajorMatrixView::new_row(preprocessed_next.as_slice()),
             ),
             partitioned_main,
-            after_challenge,
-            challenges,
+            after_challenge: vec![], // unreachable
+            challenges: &[],         // unreachable
```

Comment:
Should we just remove these fields from `DebugConstraintBuilder`?

Reply:
The folder still needs to implement some traits for `Rap` stuff so it was easier to say its empty here. Also in principle it's general enough we may want it for debugging non-interaction RAPs in the future